packages/terra-collapsible-menu-view/src/CollapsibleMenuViewItemGroup.jsx

```diff
@@ -80,7 +80,7 @@ class CollapsibleMenuViewItemGroup extends React.Component {
     } if (isCollapsibleMenuItem) {
       return (
         <li role="none">
-          <List {...customProps} role="listbox">
+          <List {...customProps} role="listbox" aria-label="Item Group">
```

Comment:
This will need a default translation and a new prop for people to customize the label if they want/need to.

Reply:
Should we add this prop and then update our examples? It seems this label should be unique to the component rendering/using it.